.nais/kafka/kafka-dev.yaml

```diff
@@ -57,3 +57,5 @@ spec:
     - access: read
       application: emottak-kafka-manager
       team: team-emottak
+
+---
```

Comment:
Hvilken funksjon har denne?

Reply:
Skillelinje for å ha flere yaml-spesifikasjoner i samme fil. Er nok bare noe som hang igjen ved et uhell når topicen som lå under linja ble flyttet ut.